modules/docker/build_test.go

```diff
@@ -60,3 +64,23 @@ func TestBuildWithTarget(t *testing.T) {
 	out := Run(t, tag, &RunOptions{Remove: true})
 	require.Contains(t, out, text1)
 }
+
+func TestGitCloneAndBuild(t *testing.T) {
```

Comment:
`t.Parallel()` was removed by intention?

Reply:
Ah good catch! Was not intentional. Added in! [443f3f8](https://github.com/gruntwork-io/terratest/pull/1062/commits/443f3f807fd24fcf28c2cdf3c1cd35831cafcf7d)